README.md

```diff
@@ -0,0 +1,126 @@
+<!-- TABLE OF CONTENTS -->
```

Comment:
Although your codebase is very precise, you could follow the DRY principle and create a function for stringify from local Storage and avoid repeating the same code. By creating a function you could simply invoke the function for the below lines. 

https://github.com/LoviMadaan/todo-review/blob/8a358df627277cbb8c3ab31c3569724238b083ac/src/todoStatus.js#L88
https://github.com/LoviMadaan/todo-review/blob/8a358df627277cbb8c3ab31c3569724238b083ac/src/todoStatus.js#L103
https://github.com/LoviMadaan/todo-review/blob/8a358df627277cbb8c3ab31c3569724238b083ac/src/todoStatus.js#L69
https://github.com/LoviMadaan/todo-review/blob/8a358df627277cbb8c3ab31c3569724238b083ac/src/todoStatus.js#L34